plugins/woocommerce-admin/client/marketplace/components/product-card/product-card.tsx

```diff
@@ -0,0 +1,120 @@
+import './product-card.scss';
+import { Product } from '../product-list-content/product-list-content';
+import { getAdminSetting } from '../../../../client/utils/admin-settings';
+
+export interface ProductCardProps {
+	type?: string;
+	product: Product;
+}
+
+function ProductCard( props: ProductCardProps ): JSX.Element {
+	const { product } = props;
+	const currencySymbol = getAdminSetting( 'currency' )?.symbol;
+
+	// Adjust the height of all product cards to match the tallest one.
+	const adjustProductCardHeights = () => {
+		const productCards = document.querySelectorAll(
+			'.woocommerce-marketplace__product-card'
+		);
+
+		let maxContentHeight = 0;
+
+		productCards.forEach( ( card ) => {
+			const content = card.querySelector(
+				'.woocommerce-marketplace__product-card__content'
+			) as HTMLElement;
+			if ( content ) {
+				maxContentHeight = Math.max(
+					maxContentHeight,
+					content.offsetHeight
+				);
+			}
+		} );
+
+		productCards.forEach( ( card ) => {
+			const container = card.querySelector(
+				'.woocommerce-marketplace__product-card__content'
+			) as HTMLElement;
+			if ( container ) {
+				container.style.height = maxContentHeight + 'px';
+			}
+		} );
+	};
+
+	useEffect( () => {
+		adjustProductCardHeights();
+
+		// Add event listener to adjust product card heights when the window is resized.
+		window.addEventListener( 'resize', adjustProductCardHeights );
+
+		// Remove the event listener when the component unmounts.
+		return () => {
+			window.removeEventListener( 'resize', adjustProductCardHeights );
+		};
+	}, [] );
+
+	return (
+		<Card className="woocommerce-marketplace__product-card">
+			<div className="woocommerce-marketplace__product-card__content">
+				<div className="woocommerce-marketplace__product-card__header">
+					<div className="woocommerce-marketplace__product-card__details">
+						{ product.icon && (
+							<img
+								className="woocommerce-marketplace__product-card__icon"
+								src={ product.icon }
+								alt={ product.title }
+							/>
+						) }
+						<div className="woocommerce-marketplace__product-card__meta">
+							<h2 className="woocommerce-marketplace__product-card__title">
+								{ product.title }
+							</h2>
+							<p className="woocommerce-marketplace__product-card__vendor">
+								<span>{ __( 'By ', 'woocommerce' ) }</span>
+								<a
+									href={ product.vendorUrl }
+									target="_blank"
+									rel="noopener noreferrer"
+								>
+									{ product.vendorName }
+								</a>
+							</p>
+						</div>
+					</div>
+				</div>
+				<p className="woocommerce-marketplace__product-card__description">
+					{ product.description }
+				</p>
+				<Button
+					className="woocommerce-marketplace__product-card__price"
+					href={ product.url }
+					target="_blank"
+					variant="link"
+				>
+					<span>
+						{ product.price === 0 || product.price === '0'
+							? __( 'Free download', 'woocommerce' )
+							: product.price + currencySymbol }
```

Comment:
The design has the currency symbol first.

```suggestion
							: currencySymbol + product.price }
```

Reply:
Thank you, I missed updating as per the new design. [Old design](2sXEw33oLyTZ7Mu3808sed-fi-1018_179513) has it at the end.